assets/models/foundrylocal/openai-whisper-base-cuda-gpu/spec.yaml

```
@@ -0,0 +1,27 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: openai-whisper-base-cuda-gpu
+version: 2
```

This should be version 1 if it hasn't been published before
```suggestion
version: 1
```